packages/x-date-pickers/src/TimeField/tests/editing.TimeField.test.tsx

```diff
@@ -77,15 +77,15 @@ describe('<TimeField /> - Editing', () => {
           format: adapter.formats.fullTime12h,
           defaultValue: adapter.date(new Date(2022, 5, 15, 0, 12, 25)),
           key: 'ArrowDown',
-          expectedValue: adapterName === 'date-fns' ? '11:12 am' : '11:12 AM',
+          expectedValue: '11:12 AM',
```

Comment:
🙌 💯 
This is an actual bug-fix IMHO. 😆 
But it also warrants at least a minor bump on it's own, because of the fact that in some cases, this could be a breaking change for some1 🙈